$OpenBSD: patch-js_src_ctypes_libffi_src_mips_ffitarget_h,v 1.2 2013/12/12 19:53:19 landry Exp $

There's no sgidefs on OpenBSD, so don't bother including it.
https://hg.mozilla.org/integration/mozilla-inbound/rev/a434e4fc3482
Remove in 28

--- js/src/ctypes/libffi/src/mips/ffitarget.h.orig	Fri Apr 15 10:03:48 2011
+++ js/src/ctypes/libffi/src/mips/ffitarget.h	Fri Apr 15 10:04:04 2011
@@ -29,7 +29,7 @@
 
 #ifdef linux
 # include <asm/sgidefs.h>
-#else
+#elif !defined(__OpenBSD__)
 # include <sgidefs.h>
 #endif
 #  ifndef _ABIN32
